dev/breeze/tests/test_build_image.py

```diff
@@ -1,60 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Those tests can be deleted now. We completely decoupled caching parameters from creating BuildProdParams, BuildCIParams - by the time they get created, the cache retrieval/writing  is already processed via `CacheableChoice` and those tests make no sense.